arithmetization/src/main/java/net/consensys/linea/zktracer/module/mod/ModOperation.java

```diff
@@ -127,8 +127,9 @@ private static BaseBytes getRes(OpCode op, Bytes32 arg1, Bytes32 arg2) {
   }
 
   private UInt256 absoluteValueIfSignedInst(Bytes32 arg) {
-    if (isSigned) {
-      return UInt256.valueOf(arg.toBigInteger().abs());
+    if (this.isSigned()) {
+      Bytes argBytes = Bytes.of(arg.toArray());
+      return UInt256.valueOf(argBytes.toBigInteger().abs());
```

Comment:
This was the culprit. See screenshots here https://github.com/Consensys/linea-tracer/issues/1180#issuecomment-2351784017

Reply:
Why not using, in the case of signed op : `return UInt256.valueOf(arg.toUnsignedBigInteger());` ?

`.toUnsignedBigInteger()` is what we use everywhere

Reply:
LGTM. We have to deal with https://github.com/Consensys/linea-tracer/issues/823 to have less conversions everiwhere ...